src/app/exercise/editors/editors.component.scss

```diff
@@ -81,10 +85,17 @@
   border: 1px solid #000;
   margin-bottom: 2px;
 }
+
 .file-ts {
   background: #1a67bf;
   color: white;
 }
-.file-html{
+
+.file-html {
   background: #9cff7d;
 }
+
+:host /deep/ .current-line {
+  display: none !important;
```

Comment:
I really don't want this thin to ever ever show up in any circumstances. 
Added a todo